src/py42/clients/alerts.py

```diff
@@ -2,6 +2,11 @@
 
 
 class AlertsClient(object):
+    """A client to expose alert API.
```

Comment:
Any reason why we do method level docs at some points and class level at other places?

Like couldn't `search()` have one of its own?

Reply:
There are multiple reasons:

- Some clients do not have method's equivalent definition in API Gateway. e.g. `create_user` in `UserClient`

- Some clients consume multiple micro services. e.g. `SecurityData`

- Some classes are still not added in API Gateway. e.g. `Devices`, `ArchiveClient`

So at this point I left some of the existing swagger docs link as is, based on my understanding.

Reply:
If you to py42docs.code42.com, start searching through the method documentation, find the method you want, you won't see the REST documentation link without scrolling around to the top of the client.

Reply:
Need to understand whether we want to remove all the swagger docs in this PR or change as and when we add other docs to API Gateway.

Reply:
Probably add other docs as they come in to the gateway

Reply:
Added rest documentation link to header considering all the links refer to the same page. As requested, added links to few (not all) method documentation to make sure user can find the link in the method if they miss out on the top of the page.